contracts/axelar-gateway/src/auth.rs

```diff
@@ -93,7 +89,11 @@ pub fn rotate_signers(env: &Env, new_signers: &WeightedSigners, enforce_rotation
         .persistent()
         .set(&DataKey::SignerHashByEpoch(new_epoch), &new_signers_hash);
 
-    // If new_signers has been rotated to before, we will overwrite the epoch to point to the latest
+    // signers must be distinct, since nonce should guarantee uniqueness even if signers are repeated
```

Comment:
also need to update gateway::rotate_signers where duplication is checked by storing a `RotationExecuted` key. The goal of this change is that we don't need to do that separately, similar to the EVM gateway

Reply:
I see. updated.